src/js/base/ffi-helpers.js

```diff
@@ -456,6 +456,8 @@ define(["js/runtime-util", "trove/lists", "trove/sets", "trove/option", "trove/e
 
         makeList: makeList,
         makeTreeSet: makeTreeSet,
+
+        isOption: runtime.getField(O, "is-Option"),
```

Comment:
I think then the `isEither` line is wrong too...  And, I guess we never call it at all :)